docs/developers/contributing/index.md

```diff
@@ -118,6 +118,13 @@ also include benchmarks to show the new feature is not too slow but this is less
 poor performance, [profiling](profiling) and [performance monitoring](napari-perfmon) can help
 identify the cause and where to optimize.
 
+### API Changes
+
+If you are changing an existing API or adding a new one, make sure you update
+the corresponding docstring to contain the `.. versionadded::` or
+`.. versionchanged::` directive. For more information see the
+[Sphinx documentation](https://www.sphinx-doc.org/en/master/usage/restructuredtext/directives.html#describing-changes-between-versions).
```

Comment:
```suggestion
[Sphinx documentation](https://www.sphinx-doc.org/en/master/usage/restructuredtext/directives.html#describing-changes-between-versions). 
Please also consider documenting any major features/changes in our
[tutorials](tutorials) and other [usage documentation](usage).
```
How about this as a middle ground? encourage but not require?
At the very least would be nice to have issues created that a new feature requires docs -- this is something we should do in review.

Reply:
Oh! I think I misunderstood @lucyleeow 's suggestion. I thought she meant adding a note about the versionadded/versionchanged directives in the "contributing docs" section, not the other way around. Sure, this is reasonable!

Reply:
I'll let @lucyleeow merge, in case I also misunderstood.

Reply:
Yup that's exactly what I was thinking, thanks everyone, I'll merge!